src/app.py

```
@@ -58,6 +64,12 @@ def get_title(title) -> str:
 
 @app.route("/api/list", methods=["POST"])
 def get_titles():
+    # Check for User-Agent header
+    if not request.headers.get('User-Agent'):
+        response_status = "User-Agent missong"
+        logs_db.log_request("/api/<title>", title, response_status, delta)
```

Ruff (0.8.2)

Undefined name `title`

(F821)